src/agent/thread_helper.cpp

```diff
@@ -292,6 +292,11 @@ void ThreadHelper::Attach(ResultHandler aHandler)
     }
 }
 
+otError ThreadHelper::Detach(void)
+{
+    return otThreadSetEnabled(mInstance, false);
```

Comment:
The "Attach" function calls the function `otIp6SetEnabled` to start the Ipv6 stack, should we  also call the `otIp6SetEnabled` to disable the ipv6 stack in the `Detach` function?

Reply:
Updated. I think both are OK, but disabling the stack may save some resources.